AdminClient/package.json

```diff
@@ -11,37 +11,42 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.2.14",
```

Comment:
good version updates, but does this mean we need a minimum version of these dependencies?

Reply:
Yes, cause of angular-cli being updated to version 9, everything angular in the admin client will be updated to version 9.